doc/command_line_tool.md

```diff
@@ -38,15 +38,27 @@ In order to build the grpc command line tool from a fresh clone of the grpc
 repository, you need to run the following command to update submodules:
 
 ```
-git submodule update --init
+$ git submodule update --init
 ```
-
+---
+### Unix user
 You also need to have the gflags library installed on your system. On Linux
 systems, gflags can be installed with the following command:
 
 ```
-sudo apt-get install libgflags-dev
+$ sudo apt-get install libgflags-dev
+```
+
+### OSX user
+This is a C++ CLI, and you need gflags to build it.
+```
+$ brew install gflags
+```
+You need to have automake installed on your system.
```

Comment:
This can be removed, as it's already covered in the installation instructions at https://github.com/grpc/grpc/blob/master/INSTALL.md, which are linked to at the start of this section as pre-requisites.

Reply:
got it. I did not see it